<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notification</class>
 <widget class="QWidget" name="Notification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>76</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>76</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 255, 254);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="addFriendWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>76</height>
    </rect>
   </property>
   <widget class="QLabel" name="userNameLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>USERNAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yesButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>5</y>
      <width>64</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="userIDLabel">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;ID&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>64</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="pureNoticeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>76</height>
    </rect>
   </property>
   <widget class="QLabel" name="noticeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>10</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>收到</string>
    </property>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>pureNoticeWidget</zorder>
  <zorder>addFriendWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
